cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -521,17 +543,26 @@ fn build_custom_cmd(name: &str, spec: &Spec) -> Result<clap::Command, Error> {
         cmd = cmd.alias(kebab_name);
     }
     let func = spec.find_function(name).unwrap();
-    let doc: &'static str = Box::leak(func.doc.to_string_lossy().into_boxed_str());
+    let doc: &'static str = Box::leak(arg_file_help(&func.doc.to_string_lossy()).into_boxed_str());
     cmd = cmd.about(Some(doc));
     for (name, type_) in inputs_map.iter() {
         let mut arg = clap::Arg::new(name);
+        let file_arg_name = fmt_arg_file_name(name);
+        let mut file_arg = clap::Arg::new(&file_arg_name);
         arg = arg
             .long(name)
             .alias(name.to_kebab_case())
             .num_args(1)
             .value_parser(clap::builder::NonEmptyStringValueParser::new())
             .long_help(spec.doc(name, type_).unwrap());
 
+        file_arg = file_arg
+            .long(&file_arg_name)
+            .alias(file_arg_name.to_kebab_case())
+            .num_args(1)
+            .hide(true)
+            .value_parser(value_parser!(PathBuf));
```

Comment:
Can we add a "conflicts-with" for the normal arg? That would clarify the handling if you provide both.

Reply:
Good point! Will do!